api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -369,10 +361,15 @@ private WorkspaceAccessLevel getWorkspaceAccessLevel(String workspaceNamespace,
   private void createConfigFile(org.pmiops.workbench.firecloud.model.Workspace fcWorkspace) {
     JSONObject config = new JSONObject();
 
+    WorkbenchConfig workbenchConfig = workbenchConfigProvider.get();
     config.put(WORKSPACE_NAMESPACE_KEY, fcWorkspace.getNamespace());
     config.put(WORKSPACE_ID_KEY, fcWorkspace.getName());
     config.put(BUCKET_NAME_KEY, fcWorkspace.getBucketName());
     config.put(API_HOST_KEY, this.apiHostName);
+    // TODO: make these based on the CDR version for the workspace; update this file if the
+    // CDR version changes.
+    config.put(CDR_VERSION_PROJECT, workbenchConfig.bigquery.projectId);
```

Comment:
Is this the right approach? These things all seem fundamentally request-scoped. Why not just pull in the CdrVersion right now? Isn't the data available to these methods?

I'm growing concerned with the number of hardcoded things / TODOs I've found relating to CdrVersion.

Reply:
CdrVersion is unfortunately just a placeholder in our schema at this point... we haven't bothered even populating CdrVersion in the database with the correct values yet.

I am growing nervous about the TODOs as well. I'd like to address them, though it may have to wait until after Feb 1.